dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -18,21 +20,26 @@ function DatePickerWidget({
   setDateRange,
 }) {
   const [fromDate, setFromDate] = useState({});
-  const [strDate, setStrDate] = useState(formatDate(new Date()));
-  const [toDate, setToDate] = useState(constructUTCDate(new Date(strDate)));
+  const [toDate, setToDate] = useState(
+    constructUTCDate(formatDate(new Date(bumpToDate(new Date(), 1))))
+  );
+  const [strDate, setStrDate] = useState(
+    new Date().toLocaleString("fr-CA").split(",")[0]
+  );
```

Comment:
French Canadian, eh?  (🤣  Sorry if you don't get the joke, Shubham.)  This warrants a comment!

Also, instead of splitting the resulting string, try using [`toLocaleDateString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString).